midealocal/devices/x13/message.py

```diff
@@ -59,32 +61,28 @@ def _body(self):
 
 
 class MessageMainLightBody(MessageBody):
-    def __init__(self, body):
+    def __init__(self, body: bytearray) -> None:
         super().__init__(body)
         self.brightness = self.read_byte(body, 1)
         self.color_temperature = self.read_byte(body, 2)
         self.effect = self.read_byte(body, 3) - 1
         if self.effect > 5:
             self.effect = 1
-        """
-        self.rgb_color = [self.read_byte(body, 5),
-                          self.read_byte(body, 6),
-                          self.read_byte(body, 7)]
-        """
         self.power = self.read_byte(body, 8) > 0
 
 
 class MessageMainLightResponseBody(MessageBody):
-    def __init__(self, body):
+    def __init__(self, body: bytearray) -> None:
         super().__init__(body)
-        self.control_success = body[1] > 0
+        self.control_success: bool = body[1] > 0
 
 
 class Message13Response(MessageResponse):
-    def __init__(self, message):
-        super().__init__(message)
+    def __init__(self, message: bytes) -> None:
+        super().__init__(bytearray(message))
         if self.body_type == 0xA4:
             self.set_body(MessageMainLightBody(super().body))
         elif self.message_type == MessageType.set and self.body_type > 0x80:
             self.set_body(MessageMainLightResponseBody(super().body))
         self.set_attr()
+        self.control_success: bool
```

Comment:
Can you please explain this? I didn't get where this came from.

Reply:
This is a variable used but not defined in init, which is a typing issue. Also the type itself was missing